libs/vkd3d/cache.c

```diff
@@ -2623,8 +2623,7 @@ void vkd3d_pipeline_cache_compat_from_state_desc(struct vkd3d_pipeline_cache_com
         HF32(desc->rasterizer_state.DepthBiasClamp);
         HF32(desc->rasterizer_state.SlopeScaledDepthBias);
         H32(desc->rasterizer_state.DepthClipEnable);
-        H32(desc->rasterizer_state.MultisampleEnable);
```

Comment:
Should these be removed?

Reply:
Or was MultisampleEnable purely line rendering related?

Reply:
These are entirely removed from RASTERIZER_DESC2 and only affect line rasterization.